node/node_test.go

```diff
@@ -198,102 +197,14 @@ func TestNodeStartMining(t *testing.T) {
 
 }
 
-// skipped anyway, now commented out.  With new mining we really need something here though.
```

Comment:
killing this. It's been sitting here for months.

Reply:
🎃 🎄 🎆